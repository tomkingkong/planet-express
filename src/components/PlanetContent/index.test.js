import React from 'react';
import { shallow } from 'enzyme';

import { PlanetContent } from '.';

describe('PlanetContent component', () => {
  it('should match snapshot', () => {
    const mockProps = {
      name: '', 
      image: '', 
      distance: 1, 
      hostStar: '', 
      yearLength: 1, 
      numInSystem: '',
      discoveryMethod: ''
    };
    const wrapper = shallow(
      <PlanetContent 
        {...mockProps}
      />);
    expect(wrapper).toMatchSnapshot();
  });
});