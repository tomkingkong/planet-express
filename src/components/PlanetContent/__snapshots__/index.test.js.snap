// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanetContent component should match snapshot 1`] = `
<article
  className="PlanetContent"
>
  <div
    className="planet__arrival"
  >
    <Component
      planetMap=""
    />
  </div>
  <div
    className="planet__info"
  >
    <h4
      className="planet__title"
    />
    <p>
       planet orbiting star 
      .
    </p>
    <p>
      1
       parsecs from Earth.
    </p>
    <p>
      1.00
       to one Earth year.
    </p>
    <p>
      Disovered using 
      .
    </p>
  </div>
</article>
`;
